test/regression/api/v3/admin/settings_spec.js

```diff
@@ -59,6 +59,7 @@ const defaultSettingsKeys = [
     'mailgun_api_key',
     'mailgun_domain',
     'mailgun_base_url',
+    'email_track_opens',
```

Comment:
Not sure why the v3 regression test is different to the v2 and canary regression tests. Perhaps some future cleanup work is needed?